.ci/test/main.sh

```diff
@@ -2,12 +2,51 @@
 
 set -ex
 
+run_NameNodeTest() {
```

Comment:
Can you generify and deduplicate some of this?

```bash
retry () {
    for i in {1..3}
    do
        $1
        if [ $? == 0 ] 
	then 
		return 0 
	fi
    done

    return 1
}

...

./ReplicationTest
./DeleteTest
retry ./NameNodeTest
./NativeFsTest
retry ./ZKDNClientTest
./ZKLockTest
retry ./ZKWrapperTest
./ReadWriteTest
```

Reply:
Good idea. This makes the code much cleaner. Changes have been pushed.